daisy_workflows/image_build/windows/post_install.ps1

```diff
@@ -359,6 +359,43 @@ function Set-Repos {
   Run-Command 'C:\ProgramData\GooGet\googet.exe' -root 'C:\ProgramData\GooGet' addrepo 'google-compute-engine-stable' 'https://packages.cloud.google.com/yuck/repos/google-compute-engine-stable'
 }
 
+function Export-ImageMetadata {
+  $version = (Get-ComputerInfo).OsVersion
```

Comment:
Make this a variable so you dont query wmi more than once

Reply:
extract as $computer_info